CHANGES.md

```diff
@@ -4,6 +4,8 @@ next (2.0.0)
 - Big refactoring by @nojb. Changes in interface (simplification of code, might
   affect performance a little). To be tested!
 
+- Lower OCaml requirement to 4.04.2  (#213, @jfrolich)
+
```

Comment:
Actually, this entry should go above the previous entry, so that we have most-recent-first. Thanks!